client/lib/wpcom-xhr-wrapper/index.js

```diff
@@ -13,7 +13,7 @@ export default function ( params, callback ) {
 		xhr.default( params, function ( error, response, headers ) {
 			if ( error && error.name === 'InvalidTokenError' ) {
 				debug( 'Invalid token error detected, authorisation probably revoked - logging out' );
-				require( 'lib/user/utils' ).logout();
+				require( 'lib/user/utils' ).default.logout();
```

Comment:
The explicit reference to `default` here seems kludgy, but I'm not exactly sure what the best course of action is. Obviously this file is a huge app-wide dependency, so I definitely want to make sure it's right before this PR gets merged.